src/drivers/imu/mpu9250/MPU9250_mag.cpp

```diff
@@ -347,3 +354,100 @@ MPU9250_mag::ak8963_setup()
 
 	return OK;
 }
+
+void MPU9250_mag::write_imu_reg_verified(int reg, uint8_t val, uint8_t mask)
+{
+	uint8_t b;
+	int retry = 5;
```

Comment:
Any idea how many retries it's hitting?

Reply:
> Any idea how many retries it's hitting?

None. It executes only once in each my test.